Связующее звено

k1808 opened the pull request:
---
:mortar_board: [Связующее звено](https://up.htmlacademy.ru/yii/1/user/201086/tasks/17)

k1808 committed:
adds messages rest

k1808 committed:
adds messages rest

Sergey-Rozhkov commented on frontend/config/main.php:
```diff
@@ -12,7 +12,10 @@
-      'gii'
+      'gii',
+      'api' => [
+        'class' => 'frontend\modules\api\Module'
```
странно что api лежит в модуле front-end

k1808 committed:
preparation REST

k1808 committed:
fixes the state of the project with a non-working api

k1808 committed:
changes the encoding of the end of lines

k1808 committed:
adds modules api end error config

k1808 committed:
fixes bugs

k1808 committed:
Merge remote-tracking branch 'origin/module8-task2' into module8-task2

# Conflicts:
#	api/config/main.php
#	api/controllers/MessagesController.php
#	api/controllers/SiteController.php
#	api/controllers/TaskController.php
#	api/controllers/UserController.php
#	api/models/Messages.php
#	frontend/config/main.php
#	frontend/models/UsersCategories.php
#	frontend/modules/api/Module.php
#	frontend/modules/api/controllers/BaseApiController.php
#	frontend/modules/api/controllers/DefaultController.php
#	frontend/modules/api/controllers/MessagesController.php
#	frontend/modules/api/models/Messages.php

k1808 committed:
fixes encoding

k1808 committed:
adds accaunt page

k1808 committed:
fixed change

k1808 committed:
fixed rest

Commit:
примерная реализация отображения сообщений в чате

Commit:
пример простого Unit теста

запуск теста: php vendor/bin/codecept run frontend/tests/unit/ExampleTest.php
инициализация: php vendor/bin/codecept generate:test unit Example -c frontend
генерация: php vendor/bin/codecept generate:test unit Example -c fronten

k1808 committed:
rest 2

k1808 committed:
Merge branch 'module8-task2' of github.com:malashenkoEvgeniy/201086-task-force-1 into module8-task2

 Conflicts:
	frontend/modules/api/controllers/MessagesController.php

k1808 committed:
method Post dont work in REST

Commit:
добавил параметр formName = ''

k1808 committed:
method Post do not work again

k1808 committed:
Merge branch 'module8-task2' of github.com:malashenkoEvgeniy/201086-task-force-1 into module8-task2

 Conflicts:
	frontend/modules/api/controllers/MessagesController.php

Commit:
сохранение сообщения из чата в БД

k1808 committed:
do not work ajax

Commit:
сохранение сообщения из чата в БД, статус код 201

k1808 committed:
added page my-list and rest for signup
